tests/test_modules.py

```diff
@@ -106,7 +106,7 @@ def test_enable_invalid(self):
             self.module_base.enable(["httpd:invalid"])
 
     def test_enable_different_stream(self):
-        self.module_base.enable(["httpd:2.4"])
+        self.base._moduleContainer.enable("httpd", "2.4", False)
```

Comment:
This is not the problem of you PR, but the problem was here before. As you can see, tests are without any description, therefore it is difficult to guess what test is testing (WHY?), what test actually do  (How?).

This test tests whether it is possible to change stream. The first part is setting of initial condition for the test - one stream is enabled. The second part is testing part. May I ask you to add some comments to the tests?